lib/Field/RepeatableSettings.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+const RepeatableSettings = ({
+  isTextDirLTR,
+  fieldIsActive,
+  repeatPosition,
+  isLastRepeat
+}) => {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
```

Comment:
Is there a better way to do left pad in JS? Or is this fine 😄

Reply:
Even if there was I wouldn't have an issue with this, it's readable!